lib/ForkProcessSandbox.cpp

```diff
@@ -35,23 +35,38 @@ pid_t mullFork(const char *processName) {
   return pid;
 }
 
-mull::ExecutionResult
-mull::ForkProcessSandbox::run(std::function<void (ExecutionResult *)> function,
-                                long long timeoutMilliseconds) {
-
-  /// Preparing pipes for child process to write to and parent process to read from.
-  int stdout_file_descriptor[2];
-  int stderr_file_descriptor[2];
-  if (pipe(stdout_file_descriptor) == -1) {
-    perror("stdout pipe");
+static std::string readFileAndUnlink(const char *filename) {
+  FILE *file = fopen(filename, "r");
+  if (!file) {
+    perror("fopen");
     exit(1);
   }
-
-  if (pipe(stderr_file_descriptor) == -1) {
-    perror("stderr_pipe");
-    exit(1);
+  if (unlink(filename) == -1) {
+    perror("unlink");
   }
 
+  fseek(file, 0, SEEK_END);
```

Comment:
Let me get this right: you first delete the file and then read from it? Is this a normal practice?

Reply:
Your method should be smth like: `unlinkFileAndRead` 😄

Reply:
The man page would explain it better than I can
https://linux.die.net/man/2/unlink

Reply:
Good.